src/pages/components/ResourceController.qml

```diff
@@ -1,6 +1,7 @@
 /****************************************************************************
 **
-** Copyright (C) 2013 Jolla Ltd.
+** Copyright (C) 2013-2019 Jolla Ltd.
```

Comment:
Could you lower-case c from (c).

We discussed once with @spiiroin that also years should be marked with spaces around the dashes. So, it should look like:

> ** Copyright (c) 2013 - 2019 Jolla Ltd.

:smile: